.github/workflows/ci.yaml

```diff
@@ -34,6 +34,29 @@ jobs:
       - name: Install dependencies
         run: yarn
 
+      - name: Compile JS with Closure
+        run: gulp dist
+
+      - name: E2E Tests
+        run: gulp e2e
+
+  e2e-with-new-build-system:
```

Comment:
Would `e2e-vite` be a little more clear?

Reply:
Yes, that's cleaner 🧼